resources/images/view/view.go

```diff
@@ -37,8 +36,8 @@ var (
 type RESTView struct {
 }
 
-func (p *RESTView) RenderSuccessPost(w rest.ResponseWriter, r *rest.Request, id string) {
-	w.Header().Add(HttpHeaderLocation, fmt.Sprintf("%s/%s", r.URL.String(), id))
```

Comment:
`r.URL.String() -> r.URL.Path`

would do the trick, if we would just align routing of gateway with service routing that should have been done :/